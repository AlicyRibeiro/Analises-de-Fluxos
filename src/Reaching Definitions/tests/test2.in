4 8
x = y + z
y = x + 1
z = y + 2
w = z + 3
x = w + 4
y = x + 5
z = y + 6
w = z + 7
1 2
2 3
3 4
0