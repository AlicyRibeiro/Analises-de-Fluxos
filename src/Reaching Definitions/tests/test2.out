Bloco 0:
  IN : { }
  OUT: { (x,0,0) (y,0,1) }
Bloco 1:
  IN : { (x,0,0) (y,0,1) }
  OUT: { (w,1,1) (x,0,0) (y,0,1) (z,1,0) }
Bloco 2:
  IN : { (w,1,1) (x,0,0) (y,0,1) (z,1,0) }
  OUT: { (w,1,1) (x,2,0) (y,2,1) (z,1,0) }
Bloco 3:
  IN : { (w,1,1) (x,0,0) (x,2,0) (y,0,1) (y,2,1) (z,1,0) }
  OUT: { (w,3,1) (x,0,0) (x,2,0) (y,0,1) (y,2,1) (z,3,0) }
